### ISSUE TITLE
Fabric is unsupported.  You read that right.

### ISSUE THREAD

**Issue by piousminion** [qouteall/ImmersivePortalsMod#841]
Can you please fix the Forge version to where it's at least usable?  It's broken(mixins version problem).   No major mods use Fabric and no one wants to dump every mod they use to play yours by itself.

e.g. https://github.com/mekanism/Mekanism/issues/7063#issuecomment-800502330

I HAVE posted on the forge version issues page about the mixins problem, but it seems like the author(s) are fabric elitists that don't even bother reading them. I imagine the response I'll get to this, but what other recourse do I have. Tell me.

**Comment by VexianEmpire** [comment]
unsupported by what? your inability to change? all forge does nowadays is make things overly complicated for mod makers and mod players, and it took very long to update to 1.17, imagine how long it'll take when 1.18 comes out, meanwhile fabric updates to even every snapshot because it isn't filled with things nobody uses that take up tons of storage space. you are just too caught up in your own nostalgia for 1.7 to see that fabric is not the worst thing ever, let alone comparable to forge. how ironic that you call us fabric elitists when you are an elitist for forge. at least we realise that forge isn't the worst modloader in existence.

also, don't post your forge issues on the fabric version. at least have some patience if you won't have any consideration for others.

**Comment by qouteall** [comment]
This mod's Forge version is MIT licensed and you can try to fix it by yourself. I didn't maintain the Forge version because maintaining it is harder than Fabric and is not interesting. This mod is provided free and no warranty is provided. I may remake the Forge version in 1.17 in the future.

**Comment by piousminion** [comment]
@qouteall 
> This mod's Forge version is MIT licensed and you can try to fix it by yourself. I didn't maintain the Forge version because maintaining it is harder than Fabric and is not interesting. This mod is provided free and no warranty is provided. I may remake the Forge version in 1.17 in the future.
That's a far more professional reply than the other one I received.  Thank you for that.  Anyway, I most certainly would try to fix it myself, but even though I do some coding despite my disability, java has been near impossible for me to wrap my head around.  I lack that talent.

I know it takes work, it's not fun/interesting, and you give it for free.  I'm not asking for a remake, or an upgrade for 1.17, it's just a version problem with mixin [issue here](https://github.com/qouteall/ImmersivePortalsModForForge/issues/284).  I'm sorry for the tone I came here with, but it's frustrating seeing it right there and just out of reach. I know I'm not entitled to anything, but it really does seem to be a lot of elitism among some of the fabric devs. I wish I didn't need forge, but for now I do. As do many other users.  :/

I'd replace the file and rebuild it myself(assuming it's that easy) if I had any idea how.  If there was an idiot proof guide to do that, I'd even TRY to tackle some of the others problems people report and make pull requests.

**Comment by piousminion** [comment]
@VexianEmpire 
> unsupported by what? your inability to change?
Unsupported by all the big mods.  All the tech mods for fabric are small.  Also, I don't make the mods so it's not me being unable to change. It's mod authors.

>  you are just too caught up in your own nostalgia for 1.7 to see that fabric is not the worst thing ever, let alone comparable to forge
Nostalgia or 1.7 has nothing to do with this and I never said Fabric was bad.  In fact, I think Fabric itself is great, but no major mods support it. Users are stuck with Forge and Fabric devs don't seem to care.

> how ironic that you call us fabric elitists when you are an elitist for forge. at least we realise that forge isn't the worst modloader in existence.
Really?  I never said that Forge was a good thing.  I simply said that it's what most mods use and most people are stuck with it until Fabric is fully embraced.  Referring to fabric authors as elitists comes from many statements equating to "fabric is the be-all end-all of modloaders. You should switch or we have nothing to talk about, peasant."  Unwillingness to consider, or even recognize what users want/need by many of these authors is a problem.

> also, don't post your forge issues on the fabric version. at least have some patience if you won't have any consideration for others.
Funny you should mention consideration for others.  The lack of consideration of users is the bulk of my complaint.  I don't intend to create additional forge-related tickets on the fabric project, but considering there is 0 indication that the forge issues were even looked at, as I said, "what recourse do I have. Tell me."

**Comment by piousminion** [comment]
@VexianEmpire 
> unsupported by what? your inability to change?

Unsupported by all the big mods.  All the tech mods for fabric are small.  Also, I don't make the mods so it's not me being unable to change. It's mod authors.

>  you are just too caught up in your own nostalgia for 1.7 to see that fabric is not the worst thing ever, let alone comparable to forge

Nostalgia or 1.7 has nothing to do with this and I never said Fabric was bad.  In fact, I think Fabric itself is great, but no major mods support it. Users are stuck with Forge and Fabric devs don't seem to care.

> how ironic that you call us fabric elitists when you are an elitist for forge. at least we realise that forge isn't the worst modloader in existence.

Really?  I never said that Forge was a good thing.  I simply said that it's what most mods use and most people are stuck with it until Fabric is fully embraced.  Referring to fabric authors as elitists comes from many statements equating to "fabric is the be-all end-all of modloaders. You should switch or we have nothing to talk about, peasant."  Unwillingness to consider, or even recognize what users want/need by many of these authors is a problem.

> also, don't post your forge issues on the fabric version. at least have some patience if you won't have any consideration for others.

Funny you should mention consideration for others.  The lack of consideration of users is the bulk of my complaint.  I don't intend to create additional forge-related tickets on the fabric project, but considering there is 0 indication that the forge issues were even looked at, as I said, "what recourse do I have. Tell me."

In short, you've made a lot of assumptions and seem to be unclear as to my complaint and the issue that many, if not most, minecraft mod users face.  You, specifically, are either oblivious to the problem, or you simply don't care...  which supports my elitist claim.  Understanding one another is better than arguing.  So, if I'm wrong, I'd like to know how, but perhaps this thread isn't the best place for that.  I'm open to suggestion as to the venue.
